src/main/java/practice/StreamPractice.java

```diff
@@ -14,7 +16,16 @@
      * "Can't get min value from list: < Here is our input 'numbers' >"
      */
     public int findMinEvenNumber(List<String> numbers) {
-        return 0;
+        if (numbers == null || numbers.size() == 0) {
+            throw new RuntimeException("Can't get min value from list:");
```

Comment:
```suggestion
        if (numbers == null || numbers.size() == 0) {
            throw new RuntimeException("Can't get min value from list:");
```
Як на мене це зайве коли працюєшь зі стрімами. Є метод [orElseThrow()](https://vertex-academy.com/tutorials/ru/java-8-optional/) для цього.

Reply:
replaced. Thanks